interview-questions.md

```diff
@@ -9,49 +9,72 @@ Then, research the question to expand on your answer. Even if you feel you have
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
 Your answer:
+A conditional statement is declared by if, else if, and else. It's contained in a code block and determines if something is true or false. Different methods can be used with primitive types of data. Booleans for example can give you true or false. Comporative statments to see if one value equal another value is also another common method. Based on the answer and desired output, it can be used to execute an action.
+
+example:
+
+// If input is "How are you"?
+// Output will return "Good"!
+
+const hello = (input) => {
+    if (input === "How are you?") {
+        return "Good!"
+    }
+}
+
+console.log(hello("How are you?")) // output: "Good!"
+
+Since "How are you?" was strictly equal to the input "How are you?", our condition was met and executed an action of retuning "Good!"
+
+
 
 Researched answer:
+A conditional statement is a programming structure in JavaScript that allows a program to make decisions based on if a condition is true or false. It allows our code to make decisions based on values given to it. Conditional statements allow our code to be more dynamic rather than run in a straight line. 
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git allows users to view changes on files and share them with others. Github is a remote console to store, share, and colloborate files with others. With the two, it allows us to pull files from our local consoles and send them to a remote console. We can build branches of code and merge them together. This is very useful when it comes to creating blocks of code that will eventually be apart of a larger program.
 
-Researched answer:
+Researched answer: Git is a version-control system for tracking changes in computer files and allow those changes to be sent to multiple people and vice versa. Its tool used by devs for source code management. It was created by Linus Torvalds in 2005 to allow other developers to contribute to Linux Kernel. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Comparison operators will return a Boolean value, which is either true or false. The comparison operators are =,==,===,!,!==,<,>,<=, and =<. All these operators will either have a true or false output.
 
-Researched answer:
+Researched answer: JavaScript operators that will return a Boolean value are comparison operators. Comparison operators like Boolean provide binary options and one answer. This is also known as a logical value, which can be viewed as true or false. 
```

Comment:
This is a great place to try to find something you don't already know about the question and broaden your knowledge. What about logical operators?